src/errors/errorPages.js

```diff
@@ -21,6 +23,9 @@ class ErrorPages {
         // eslint-disable-next-line no-unused-vars
         app.use((errors, req, res, next) => {
           const serverError = opts.serverError || {};
+
+          logger.error(errors);
```

Comment:
@michaeldfallen this doesn't seem to show up in the console for `start-dev`, is there a configuration reason for this